src/SAML2/AuthnRequestFactory.php

```diff
@@ -145,7 +146,7 @@ public static function createNewRequest(ServiceProvider $serviceProvider, Identi
      * @param SAML2_Configuration_PrivateKey $key
      * @return SAML2_Configuration_PrivateKey|XMLSecurityKey
      */
-    private static function loadPrivateKey(SAML2_Configuration_PrivateKey $key)
+    public static function loadPrivateKey(SAML2_Configuration_PrivateKey $key)
```

Comment:
Looks like you used this during testing once upon a time, but it doesn't seem like this method needs to be public.